utils/hash.go

```diff
@@ -14,27 +14,21 @@
    limitations under the License.
 */
 
-package compose
+package utils
 
 import (
 	"encoding/json"
 
+	"github.com/compose-spec/compose-go/types"
 	"github.com/opencontainers/go-digest"
 )
 
-func jsonHash(o interface{}) (string, error) {
+// ServiceHash compute configuration has for a service
+// TODO move this to compose-go
+func ServiceHash(o types.ServiceConfig) (string, error) {
 	bytes, err := json.Marshal(o)
 	if err != nil {
 		return "", err
 	}
-	return digest.SHA256.FromBytes(bytes).String(), nil
-}
-
-func contains(slice []string, item string) bool {
-	for _, v := range slice {
-		if v == item {
-			return true
-		}
-	}
-	return false
+	return digest.SHA256.FromBytes(bytes).Hex(), nil
```

Comment:
why this change to Hex ? 
And then I just noticed `// Hex is deprecated. Please use Digest.Encoded.`

Reply:
updated to use Encoded()
String() do include the algorithm (`sha256:...`), so aligned with docker-compose